tests/foreman/cli/test_system_registration.py

```diff
@@ -7,96 +7,58 @@ class SystemRegistrationTestCase(CLITestCase):
     """Tests for System Registration CLI"""
 
     @stubbed()
-    def test_rpm_install(self):
```

Comment:
this test is removed since test_contenthost.py already has several test to install katello cert